Source/DafnyCore/AST/Expressions/Applications/MemberSelectExpr.cs

```diff
@@ -215,7 +215,6 @@ public MemberSelectExpr(Cloner cloner, MemberSelectExpr original) : base(cloner,
     }
   }
 
-  [SyntaxConstructor]
```

Comment:
Why remove this?

Reply:
It's a class that's created during resolution. To create it before resolution is an error.

Reply:
I see, thanks for the explanation. It'd be great if we could prevent resolution-only syntax types from being added to the schema in the first place, since they effectively become part of the schema "API" and removing them can break consumers.